compiler/circle-eval-diff/src/ModuleEvalDiff.cpp

```diff
@@ -200,6 +213,22 @@ void H5InputEvalDiff::evalDiff(const std::string &first_input_data_path,
 
       // Accumulate diffs
       accumulateMetrics(first_output, second_output);
+
+      // Save outputs
+      for (uint32_t i = 0; i < first_output.size(); i++)
+      {
+        auto out = first_output[i];
+        writeDataToFile(output_prefix + "." + std::to_string(data_idx) + ".first.output" +
+                          std::to_string(i),
+                        (char *)(out->buffer()), out->byte_size());
+      }
+      for (uint32_t i = 0; i < second_output.size(); i++)
+      {
+        auto out = second_output[i];
+        writeDataToFile(output_prefix + "." + std::to_string(data_idx) + ".second.output" +
+                          std::to_string(i),
+                        (char *)(out->buffer()), out->byte_size());
+      }
```

Comment:
Just for note. This may change when https://github.com/Samsung/ONE/pull/8830 is landed.